### PR TITLE
Kelsonic 17514 create clp template + graphql query boilerplate

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/17514

This PR adds the full graphql query for CLP and a boilerplate for the template so that we can work on building out the template.

## Testing done
Locally

## Screenshots
image

## Acceptance criteria
- [x] adds the full graphql query for CLP and a boilerplate for the template so that we can work on building out the template.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by kelsonic** [comment]
I didn't mean to open this yet, please ignore 🙂

File: src/site/stages/build/drupal/graphql/nodeCampaignLandingPage.graphql.js [hunk 1]
```diff
@@ -0,0 +1,594 @@
...
+
+module.exports = `
+  fragment nodeCampaignLandingPage on NodeCampaignLandingPage {
+    ${entityElementsFromPages}
+    changed
+    entityId
+    title
+    fieldAdministration {
+      entity {
+        ... on TaxonomyTermAdministration {
```
Comment (kelsonic) [inline src/site/stages/build/drupal/graphql/nodeCampaignLandingPage.graphql.js:11]: should totally use fragments here, just haven't yet

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          <div class="usa-alert usa-alert-info">
+            <div class="usa-alert-body">
+              <p class="usa-alert-text">You are viewing a draft.</p>
+            </div>
+          </div>
+        {% endif %}
+
+        <article class="usa-content">
+          <!-- Title -->
+          <h1>{{ title }}</h1>
```
Comment (kelsonic) [inline src/site/layouts/campaign_landing_page.drupal.liquid:21]: This is just a boilerplate template that we can build off

File: src/site/stages/build/drupal/graphql/nodeCampaignLandingPage.graphql.js [hunk 3]
```diff
@@ -0,0 +1,594 @@
...
+        ... on ParagraphButton {
+          fieldButtonLabel
+          fieldButtonLink {
+            uri
+            title
+          }
+        }
+      }
+    }
+  }
```
Comment (ncksllvn) [inline src/site/stages/build/drupal/graphql/nodeCampaignLandingPage.graphql.js:593]: I think many of these properties could be replaced by including the paragraph fragments instead.

**Review by ncksllvn — APPROVED** [review]
Great job!

File: src/site/stages/build/drupal/graphql/nodeCampaignLandingPage.graphql.js (hunk 3 above)
Comment (kelsonic) [reply src/site/stages/build/drupal/graphql/nodeCampaignLandingPage.graphql.js:593]: https://github.com/department-of-veterans-affairs/vets-website/pull/15648#discussion_r557608725 totally 😎

**Comment by kelsonic** [comment]
@department-of-veterans-affairs/frontend-review-group ready for review! 🎉

**Review by U-DON — APPROVED** [review]

**Pull request merged**